agent/engine/engine_windows_integ_test.go

```diff
@@ -66,6 +66,12 @@ func createTestContainer() *apicontainer.Container {
 	}
 }
 
+// getLongRunningCommand returns the command that keeps the container running for the container
+// that uses the default integ test image (microsoft/windowsservercore for windows)
+func getLongRunningCommand() []string {
+	return []string{"ping", "-t", "localhost"}
```

Comment:
Is ping a long running command on Windows?

Reply:
From the doc [here](https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/ping), "-t" specifies that ping continue sending echo Request messages to the destination until interrupted. Without the flag it only sends 4 packets.